examples/multi_az_cluster/main.tf

```diff
@@ -0,0 +1,50 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  name   = "multiaz-postgresql-cluster"
+  region = "eu-central-1"
+
+  tags = {
+    Name = local.name
+  }
+}
+
+module "rds_cluster_test" {
+  source                       = "../../"
+  is_cluster                   = true
+  environment                  = "dev"
```

Comment:
The variable instance_is_multi_az is null by default, but environment is dev, so the logic in locals.tf says that local.instance_is_multi_az is false. Which means the multi az example isn't multi az afterall?

Should it not be set to instance_is_multi_az=true here?

Reply:
The example is multiaz-cluster not multiaz-instance.